### ISSUE TITLE
Could option UCI_Variant be added to NewGG?

### ISSUE THREAD

**Issue by kappa-wingman** [leedavid/NewGG#7]
Greetings.

I am able to compile NewGG on Linux.
On Linux, I am using these two GUI: Xboard and https://github.com/EterCyber/Sylvan/

For Xboard, it could set UCI but NewGG do not have the option 'UCI_Variant'
The value should be "xiangqi"

Xboard could load the NewGG game engine but display an international chess board.
So if the UCI_Variant option is added, I think it could be run with Xboard.

**Comment by Klamath233** [comment]
Yes. It would be very nice if we can make Xboard load NewGG. I'll have a look after I'm back from vacation.
